cmd/artifact/install/install.go

```diff
@@ -128,6 +144,8 @@ func NewArtifactInstallCmd(ctx context.Context, opt *options.CommonOptions) *cob
 		"directory where to install rules. Defaults to /etc/falco")
 	cmd.Flags().StringVarP(&o.pluginsDir, "plugins-dir", "", config.PluginsDir,
 		"directory where to install plugins. Defaults to /usr/share/falco/plugins")
+	cmd.Flags().Var(&o.allowedTypes, "allowed-types",
+		"whitelist of artifacts type that can be installed. If an empty list is passed (or configured), everything is allowed")
```

Comment:
```suggestion
		"comma-separated list of artifact types that can be installed. If not specified, all types are allowed. Example: \"rulesfile,plugin\"")
```

Reply:
I think this phrasing is a little clearer but the flag name and everything as implemented makes sense to me :+1:

Reply:
Agreed but it isn't only a comma-separated list, you can also specify the flag multiple times. Any other hint?

Reply:
Thanks @loresuso , I think your solution is the best since it explains all ways to use the flag!

Reply:
👍 updated!